crypto/dynamic_loading_test.c

```diff
@@ -113,7 +113,7 @@ int main(int argc, char *argv[]) {
 #else
 
 int main(int argc, char **argv) {
-  fprintf(stdout, "PASS\n");
+  fprintf(stdout, "PASS\n"); // NOLINT(clang-analyzer-security.insecureAPI.DeprecatedOrUnsafeBufferHandling)
```

Comment:
I actually don't know why this line was flagged:
```
/Users/justsmth/repos/aws-lc/crypto/dynamic_loading_test.c:116:3: warning: Call to function 'fprintf' is insecure as it does not provide security checks introduced in the C11 standard. Replace with analogous functions that support length arguments or provides boundary checks such as 'fprintf_s' in case of C11 [clang-analyzer-security.insecureAPI.DeprecatedOrUnsafeBufferHandling]
  116 |   fprintf(stdout, "PASS\n");
      |   ^~~~~~~
```

Reply:
BTW: I looked into this, but GLIBC doesn't support these secure variants (like `fprintf_s`): https://sourceware.org/glibc/manual/latest/html_node/Formatted-Output-Functions.html

It seems to mostly be an MSVC extension that's (only optionally) part of the C11 standard.